factcast-store/src/main/java/org/factcast/store/internal/catchup/fetching/PgFetchingCatchUpFactory.java

```diff
@@ -58,26 +56,12 @@ public PgFetchingCatchUpFactory(
   }
 
   @Override
-  public PgFetchingCatchup create(
+  public PgCatchup create(
```

Comment:
I think the pool & Autoclosable can go away here right?

Reply:
well, the reason for this was to shutdown the pool on system shutdown iirc.
could well be that we'd fail on that aspect. i'll better inject the pool instead and close it here.